packages/wix-ui-codemod/src/wix-ui-tpa/import-components-via-index-file.ts

```diff
@@ -0,0 +1,60 @@
+import { Transform, ImportSpecifier } from 'jscodeshift';
+
+const transformComponentsImports: Transform = (fileInfo, api) => {
+  const j = api.jscodeshift;
+  const namedImports: Set<string> = new Set();
+  const root = j(fileInfo.source);
+  const importMatch = /wix-ui-tpa/;
+  const testkitsImportMatch = /wix-ui-tpa\/dist\/src\/testkit/;
+  const stylesImportMatch = /st.css/;
+  const driversImportMatch = /.driver/;
```

Comment:
Note that in this regex the `.` means "any character" and not a real `.`.
This should probably be escaped:
```suggestion
  const driversImportMatch = /\.driver/;
```